### PR TITLE
Ida/development

### PR DESCRIPTION
**Pull request by idali0226**

### CODE REVIEW

**Comment by Inkimar** [comment]
# [Codecov](https://codecov.io/gh/Naturhistoriska/Georg/pull/3?src=pr&el=h1) Report
> Merging [#3](https://codecov.io/gh/Naturhistoriska/Georg/pull/3?src=pr&el=desc) into [master](https://codecov.io/gh/Naturhistoriska/Georg/commit/bcefb5f28659638ca292eb4e03315117025a7ccc&el=desc) will **not change** coverage by `%`.
> The diff coverage is `n/a`.

[Impacted file tree graph](https://codecov.io/gh/Naturhistoriska/Georg/pull/3?src=pr&el=tree)

```diff
@@          Coverage Diff           @@
##           master      #3   +/-   ##
======================================
  Coverage    0.00%   0.00%           
======================================
  Files           6       6           
  Lines          68      68           
  Branches        2       2           
======================================
  Misses         68      68           
```



------

[Continue to review full report at Codecov](https://codecov.io/gh/Naturhistoriska/Georg/pull/3?src=pr&el=continue).
> **Legend** - [Click here to learn more](https://docs.codecov.io/docs/codecov-delta)
> `Δ = absolute <relative> (impact)`, `ø = not affected`, `? = missing data`
> Powered by [Codecov](https://codecov.io/gh/Naturhistoriska/Georg/pull/3?src=pr&el=footer). Last update [bcefb5f...77c7923](https://codecov.io/gh/Naturhistoriska/Georg/pull/3?src=pr&el=lastupdated). Read the [comment docs](https://docs.codecov.io/docs/pull-request-comments).

File: georg-ui/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+yarn build
+```
+
+### Lints and fixes files
+```
+yarn lint
+```
+
+### Customize configuration
+See [Configuration Reference](https://cli.vuejs.org/config/).
```
Comment (jmenglund) [inline georg-ui/README.md]: Probably better with a more descriptive text for the link:
`See [Vue CLI configuration reference](https://cli.vuejs.org/config/).`

**Review by jmenglund — COMMENTED** [review]
Looks fine! Just added one small remark.

**Review by jmenglund — COMMENTED** [review]
OK!

**Pull request merged**